app/scripts/polling_location_finder.js

```diff
@@ -78,8 +86,101 @@ define(['jquery', 'geojson', 'json!vendor/ELECTIONS_WardsPrecincts.geojson', 'js
         return encodeURI(url + destination);
     }
 
-    return function(latLng, successCallback, errorCallback) {
+    function getHours(place) {
+
+        var hours = [];
+        moment.tz.add('America/New_York|EST EDT|50 40|0101|1Lz50 1zb0 Op0');
+        
+        jQuery.each(place, function(index, timeInterval) {
+            var range = moment.range(timeInterval);
```

Comment:
Haha oh well! This is super unimportant, but I'm curious to toy around with it at some point... or just move away from RequireJS altogether...